packages/dds/tree/src/simple-tree/arrayNode.ts

```diff
@@ -850,6 +854,23 @@ abstract class CustomArrayNodeBase<const T extends ImplicitAllowedTypes>
 			destinationFieldPath,
 			index,
 		);
+		this.#generationNumber += 1;
```

Comment:
this is unneeded. Performing an edit sends a change event.
```suggestion
```
```suggestion
		this.#generationNumber += 1;
```

Reply:
Updated with suggested change